src/Collection/CollectionBuilder.php

```diff
@@ -71,6 +71,21 @@ public function __construct($commandFile)
     }
 
     /**
+     * @param int $logLevel
+     *
+     * @return $this
+     */
+    public function setLogLevel($logLevel)
```

Comment:
I tried removing this from CollectionBuilder but got the error "[error]  Method setLogLevel does not exist."